markdown/ch5.markdown

```diff
@@ -430,32 +430,27 @@ print('The CSR and NumPy arrays are equal: ',
       np.all(s2 == csr.A))
 ```
 
-The ability to store large, sparse matrices is incredibly powerful!
-The combination of sparsity and linear algebra abounds.  For example,
+The ability to store large, sparse matrices, and perform computations on them,
+is incredibly powerful!
+
+The combination of sparsity and linear algebra abounds. For example,
 one can think of the entire web as a large, sparse, $N \times N$ matrix.
 Each entry $X_{ij}$ indicates whether web page $i$ links to page $j$.
 By normalizing this matrix and solving for its dominant eigenvector,
 one obtains the so-called PageRank—one of the numbers Google uses to
 order your search results. (You can read more about this in the next chapter!)
 
-Now, consider studying the human brain, and represent it as a large $M
-\times M$ graph, where there are $M$ nodes (positions) in which you
+As another example, we can represent the human brain as a large $m \times m$
+graph, where there are $m$ nodes (positions) in which you
```

Comment:
I still think it would be worth saying something here about edge == correlation strength.